upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -232,7 +232,7 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 	}
 
 	kubeDNS := b.cluster.Spec.KubeDNS
-	if kubeDNS.Provider == "KubeDNS" || kubeDNS.Provider == "" {
```

Comment:
We discussed this during office hours yesterday and determined that it will be best to default to CoreDNS only for clusters >= kubernetes 1.18. This way way upgrading kops by itself will not change the default, only upgrading kubernetes itself.

Heres an example of how you could get that information: 

https://github.com/kubernetes/kops/blob/e29a04e5af5208311b16ed8ab05234daaf97fc46/upup/pkg/fi/cloudup/apply_cluster.go#L282-L289

Reply:
@rifelpet okay, I'll update the PR to reflect these changes.

Reply:
How is `kv, err := k8sversion.Parse(cluster.Spec.KubernetesVersion); kv.IsGTE` vs `b.cluster.IsKubernetesGTE`?
I tend to prefer the latter.